Anytype/Sources/PresentationLayer/Modules/HomeWidgets/Container/HomeWidgetsModuleOutput.swift

```diff
@@ -2,5 +2,4 @@ import Foundation
 
 @MainActor
 protocol HomeWidgetsModuleOutput: AnyObject {
```

Comment:
do you need this protocol for the future?

Reply:
Used it many times in the development of new features, for temporary buttons. I want to keep it.😀

Reply:
ok, no problem)